config/tutorials/hdfs-logs/searcher-1.yaml

```diff
@@ -1,8 +1,7 @@
 version: 0
 node_id: searcher-1
-searcher:
-  rest_listen_address: 127.0.0.1
-  rest_listen_port: 7280
-  peer_seeds:
-    - 127.0.0.1:7290 # searcher-2
-    - 127.0.0.1:7300 # searcher-3
+listen_address: 127.0.0.1
+rest_listen_port: 7280
+seeds:
+  - 127.0.0.1:7290 # searcher-2
```

Comment:
As you suggested yesterday, we can make it a good practice to specify less seed node (one in this case). we can document to add more seeds as the cluster gets larger thus reducing the likelihood of try to contact a seed that is down.

Reply:
good point

Reply:
Yes, I think we should demonstrate best practices whenever possible. Best practice for adding peer seeds is the more the merrier.

Reply:
ok let's keep it as is